pylint/checkers/nested_min_max.py

```diff
@@ -0,0 +1,82 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+"""Check for use of nested min/max functions."""
+
+from __future__ import annotations
+
+import copy
+from typing import TYPE_CHECKING
+
+from astroid import nodes
+
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import only_required_for_messages, safe_infer
+from pylint.interfaces import INFERENCE
+
+if TYPE_CHECKING:
+    from pylint.lint import PyLinter
+
+
+class NestedMinMaxChecker(BaseChecker):
```

Comment:
```suggestion
class NestedMinMaxChecker(BaseChecker):
    """Multiple nested min on the same line will raise multiple message by design. The check is faster this way and the multiple messages aren't wrong."''
```

Reply:
```suggestion
class NestedMinMaxChecker(BaseChecker):
    """Multiple nested min/max calls on the same line will raise multiple messages.
    
    This behaviour is intended as it would slow down the checker to check
    for nested call with minimal benefits.
    """
```